### PR TITLE
Added the RemainingCharacters widget documentation

### PR DESCRIPTION
**Pull request by serhiyzhovnir**
## Purpose of this pull request

This pull request (PR) adds the documentation for the RemainingCharacters widget.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget-remaining-characters.html
- https://devdocs.magento.com/guides/v2.2/javascript-dev-guide/widgets/widget-remaining-characters.html

## Links to Magento source code

- https://github.com/magento/magento2/blob/2.2/app/code/Magento/Catalog/view/frontend/web/js/product/remaining-characters.js
- https://github.com/magento/magento2/blob/2.3/app/code/Magento/Catalog/view/frontend/web/js/product/remaining-characters.js

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/javascript-dev-guide/widgets/widget-remaining-characters.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+The class that appends to the [counter element](#counterselector) if the field value is empty.
+
+**Type**: String
+
+**Default value**: `'no-display'`
+
+### `noteSelector`
+
+The selector of note element.
```
Comment (rogyar) [inline guides/v2.2/javascript-dev-guide/widgets/widget-remaining-characters.md]: Hi @serhiyzhovnir. Thank you for the great topic!
Could you add more info about the note element, please? At first glance, it's not completely clear what's the note component and its main purpose.

Thank you!

**Comment by serhiyzhovnir** [comment]
Hi @rogyar 
I've improved the description for the `noteSelector` option.
Please, check it again and let me know if anything.
Thank you!

**Comment by rogyar** [comment]
Awesome, thank you!

**Comment by dobooth** [comment]
running tests

**Pull request merged**